packages/utils/src/lib/reports/generate-md-report.unit.test.ts

```diff
@@ -355,17 +358,19 @@ describe('auditsSection', () => {
           audits: [
             {
               details: {
-                issues: [{ source: {} }],
-                table: { rows: [{ value: 42 }] },
+                issues: [{ severity: 'error' }] as Issue[],
+                table: { rows: [{ value: 42 }] } as Table,
               },
             },
           ],
         },
       ],
-    } as unknown as ScoredReport);
+    } as ScoredReport).toString();
     expect(md).toMatch('#### Issues');
-    expect(md).toMatch('|Severity|Message|Source file|Line(s)|');
-    expect(md).toMatch('|value|');
+    expect(md).toMatch(
+      /\|\s*Severity\s*\|\s*Message\s*\|\s*Source file\s*\|\s*Line\(s\)\s*\|/,
```

Comment:
🤔 Might be worth creating a utility function that removes all the whitespaces.

Reply:
Wonderful idea!

Similar issues:
- #552 
- #744

Reply:
Done :heavy_check_mark: 
- #757